rpcs3/Emu/RSX/Common/texture_cache_helpers.h

```diff
@@ -169,8 +169,8 @@ namespace rsx
 			bool src_is_render_target,
 			address_range dst_range,
 			u32 dst_pitch,
-			const sizeu src_dimensions,
-			const sizeu dst_dimensions)
+			const sizeu& src_dimensions,
```

Comment:
This is intentional, pass by copy generated better bytecode. I was surprised by this as well, but apparently simple types are properly passed via registers.

Reply:
Example https://godbolt.org/z/79xEj3

Reply:
Fix it.
https://github.com/RPCS3/rpcs3/pull/9346/commits/f7c2dcd6ac1e185d2745ab681e12225f8ea2f9d5